src/doc/rustc/src/codegen-options/index.md

```diff
@@ -98,6 +98,17 @@ values:
 The default behaviour, if frame pointers are not force-enabled, depends on the
 target.
 
+## force-unwind-tables
+
+This flag forces the generation of unwind tables. It takes one of the following
+values:
+
+* `y`, `yes`, `on`, or no value: Unwind tables are forced to be generated.
+* `n`, `no`, or `off`: Unwind tables are not forced to be generated. In some
+  cases, unwind tables will still be generated if required for correctness.
```

Comment:
I think this sentence is no longer accurate, an error is generated instead.

Reply:
Updated!